### PR TITLE
pfc_gen: add the flexiblity to defer storm start and stop at Arista fanout 

### PR DESCRIPTION
**Pull request by wendani**
Infrastructure change taken out of #837; Incremental change on top of https://github.com/Azure/sonic-mgmt/pull/1134

Tested on regular pfc watchdog without break.

Signed-off-by: Wenda Ni wenni@microsoft.com

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [] Bug fix
- [] Testbed and Framework(new/improvement)
- [] Test case(new/improvement)

### CODE REVIEW

File: ansible/roles/test/templates/pfc_storm_stop_arista.j2 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 bash
 cd /mnt/flash
-sudo pkill -f {{pfc_gen_file}}
+{% if pfc_storm_stop_defer_time is defined %} sleep {{pfc_storm_stop_defer_time}} &&{% endif %} sudo pkill -f "sudo python {{pfc_gen_file}} -p {{(1).__lshift__(pfc_queue_index)}} -t 65535 -n {{pfc_frames_number}} -i {{pfc_fanout_interface | replace("Ethernet", "et") | replace("/", "_")}} -r {{ansible_eth0_ipv4_addr}}" &
```
Comment (neethajohn) [inline ansible/roles/test/templates/pfc_storm_stop_arista.j2]: why is the pkill command different here? is it intentional?

Comment (wendani) [reply ansible/roles/test/templates/pfc_storm_stop_arista.j2]: It is incremental change on top of #1134

Comment (neethajohn) [reply ansible/roles/test/templates/pfc_storm_stop_arista.j2]: why is pkill being sent to the background?

File: ansible/roles/test/templates/pfc_storm_stop_arista.j2 [hunk 2]
```diff
@@ -1,5 +1,5 @@
 bash
 cd /mnt/flash
-sudo pkill -f "sudo python {{pfc_gen_file}} -p {{(1).__lshift__(pfc_queue_index)}} -t 65535 -n {{pfc_frames_number}} -i {{pfc_fanout_interface | replace("Ethernet", "et") | replace("/", "_")}} -r {{ansible_eth0_ipv4_addr}}"
+{% if pfc_storm_stop_defer_time is defined %} sleep {{pfc_storm_stop_defer_time}} &&{% endif %} sudo pkill -f "sudo python {{pfc_gen_file}} -p {{(1).__lshift__(pfc_queue_index)}} -t 65535 -n {{pfc_frames_number}} -i {{pfc_fanout_interface | replace("Ethernet", "et") | replace("/", "_")}} -r {{ansible_eth0_ipv4_addr}}" &
```
Comment (neethajohn) [inline ansible/roles/test/templates/pfc_storm_stop_arista.j2]: why is pkill being sent to the background?

Comment (neethajohn) [reply ansible/roles/test/templates/pfc_storm_stop_arista.j2]: discussed offline.

Comment (wendani) [reply ansible/roles/test/templates/pfc_storm_stop_arista.j2]: Done

**Pull request merged**